graspologic/match/gmp.py

```diff
@@ -177,8 +179,10 @@ def fit(self, A, B, seeds_A=[], seeds_B=[]):
 
         seeds_B : 1d-array, shape (m , 1) where m <= number of nodes (default = [])
             An array where each entry is an index of a node in `B` The elements of
-            ``seeds_A`` and ``seeds_B`` are vertices which are known to be matched, that is,
-            ``seeds_A[i]`` is matched to vertex ``seeds_B[i]``.
+            `seeds_A` and `seeds_B` are vertices which are known to be matched, that is,
+            `seeds_A[i]` is matched to vertex `seeds_B[i]`.
+        S : 2d-array, square
+            A square similarity matrix
```

Comment:
can you provide more info here? also, does the "scale" of this matrix (i.e. frob norm) matter? id like to know that in the docs

Reply:
size should be the number of nonseeded nodes, so I'll add that to tests and docs. Don't think the norm needs to be restricted.

Reply:
im not saying it needs to be restricted. but i want to know if i multiply S by 2 does that affect the optimization. I think yes?

Reply:
ok i get what you mean now. so if you want both terms to be weighted the same norm(A)*norm(B) should be equal to norm(S)?

Reply:
i guess im not saying we need to do anything about this, just that we should point this out to the user

Reply:
something like: "Note, the scale of `S` may effect the weight placed on the term :math:`\\text{trace}(S^T P)` relative to :math:`\\text{trace}(A^T PBP^T)` during the optimization process."

Reply:
works for me